maptivist/src/main/webapp/index.html

```diff
@@ -90,8 +90,8 @@ <h1>Fill in Marker</h1>
                 <br>
                 <input type="text" placeholder="Enter link here" id="marker-link" name="marker-link">
                 <br>
-
-                <button type="button" class="btn" onclick="createMarker()">Submit</button>
+                <input type="submit" value="Submit">
+                <!-- <button type="button" class="btn" onclick="createMarker()">Submit</button> -->
```

Comment:
Delete unless used.

But now none of the buttons will call createMarker(). Is that intended?